pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1beta1/StatefulSet/customizations.yaml

```diff
@@ -24,24 +24,48 @@ spec:
     statusAggregation:
       luaScript: >
         function AggregateStatus(desiredObj, statusItems)
-          if statusItems == nil then
-            return desiredObj
-          end
           if desiredObj.status == nil then
             desiredObj.status = {}
           end
           if desiredObj.metadata.generation == nil then
             desiredObj.metadata.generation = 0
           end
-          generation = desiredObj.metadata.generation
-          replicas = 0
-          readyReplicas = 0 
-          currentReplicas = 0
-          updatedReplicas = 0
-          availableReplicas = 0
-          updatedReadyReplicas = 0
-          updateRevision = ''
-          currentRevision = ''
+          if desiredObj.status.observedGeneration == nil then
+            desiredObj.status.observedGeneration = 0
+          end
+        
+          -- Initialize status fields if status doest not exist
+          -- If the StatefulSet is not spread to any cluster, its status also should be aggregated
+          if statusItems == nil then
+            desiredObj.status.observedGeneration = desiredObj.metadata.generation 
+            desiredObj.status.replicas = 0
+            desiredObj.status.readyReplicas = 0
+            desiredObj.status.currentReplicas = 0
+            desiredObj.status.updatedReplicas = 0
+            desiredObj.status.availableReplicas = 0
+            desiredObj.status.updatedReadyReplicas = 0
+            desiredObj.status.expectedUpdatedReplicas = 0
```

Comment:
This field does not appear to be available in the Advanced Statefulset status: https://github.com/openkruise/kruise/blob/8ae13b1b818f4b46f9317a0ce8c806bcfe4fdc00/apis/apps/v1beta1/statefulset_types.go#L267